velox/functions/prestosql/aggregates/tests/AverageAggregationTest.cpp

```diff
@@ -29,6 +34,19 @@ class AverageAggregationTest : public AggregationTestBase {
     allowInputShuffle();
   }
 
+  core::PlanNodePtr createAvgAggPlanNode(
```

Comment:
Do we still need this?

Reply:
I have one test which runs simple aggregation to show the impact of double precision with different counts:
https://github.com/facebookincubator/velox/pull/2883/files#diff-854b68c1c9af4e7f75457fc7a873bf558b00dfbfea058fa14bb7f9af9bfdb4fdR329